criu/namespaces.c

```diff
@@ -526,7 +526,7 @@ static int open_ns_fd(struct file_desc *d, int *new_fd)
 	struct ns_desc *nd = NULL;
 	struct ns_id *ns;
 	int nsfd_id, fd;
-	char path[64];
+	char path[64] = "";
```

Comment:
I think it would be better to print a proper error if fdstore_get fails and remove the check_open label.